src/rdo.rs

```diff
@@ -240,41 +242,51 @@ pub fn rdo_mode_decision(
     };
 
     // Find the best chroma prediction mode for the current luma prediction mode
-    let cfl = &CFLParams::new();
     for &chroma_mode in &mode_set_chroma {
+      let mut cfl_alpha_set = vec![ [-1, 0], [-1, 1], [-1, -1], [1, -1], [0, -1], [1, 0], [0, 1], [1, 1] ];
+      if chroma_mode != PredictionMode::UV_CFL_PRED {
+        cfl_alpha_set.truncate(1);
```

Comment:
Oh, is this doing cfl_alpha_set become null, if current chroma_mode is not CFL? So that below 
 for &[alpha_u, alpha_v] in &cfl_alpha_set {.... } does not run?

Reply:
I checked that cfl_alpha_set.truncate(1);  makes the size  of cfl_alpha_set  as 1, so still have one entry then
the below for &[alpha_u, alpha_v] in &cfl_alpha_set { ... " is iterated once.

Reply:
truncate(0) here?